app/controllers/api/v0/order_cycles_controller.rb

```diff
@@ -70,22 +70,7 @@ def producer_properties
       end
 
       def search_params
-        permitted_search_params = params.slice :q, :page, :per_page
-
-        if permitted_search_params.key? :q
-          permitted_search_params[:q].slice!(*permitted_ransack_params)
-        end
-
-        permitted_search_params
-      end
-
-      def permitted_ransack_params
-        [
-          "#{[:name, :meta_keywords, :variants_display_as,
-              :variants_display_name, :supplier_name]
-          .join('_or_')}_cont",
-          :with_properties, :primary_taxon_id_in_any
-        ]
+        params.slice :q, :page, :per_page
```

Comment:
It would be good to know why this filtering is now unnecessary.

Reply:
There's a whitelisting of params that are used by Ransack which takes place automatically, and the allowed attributes and scopes are defined on models, eg:

https://github.com/openfoodfoundation/openfoodnetwork/blob/a096c56658bd9dd473898b3970bca8acc16b389e/app/models/spree/product.rb#L36-L38

So this big chunk of messy code is redundant; it's doing something that's already being done elsewhere.

Reply:
Hooray!